go.mod

```diff
@@ -50,20 +44,22 @@ require (
 	golang.org/x/sync v0.16.0
 	golang.org/x/term v0.34.0
 	golang.org/x/text v0.28.0
-	k8s.io/api v0.33.4
-	k8s.io/apiextensions-apiserver v0.33.4
-	k8s.io/apimachinery v0.33.4
-	k8s.io/apiserver v0.33.4
-	k8s.io/client-go v0.33.4
-	k8s.io/component-base v0.33.4
+	k8s.io/api v0.34.1
+	k8s.io/apiextensions-apiserver v0.34.1
+	k8s.io/apimachinery v0.34.1
+	k8s.io/apiserver v0.34.1
+	k8s.io/client-go v0.34.1
+	k8s.io/component-base v0.34.1
 	k8s.io/gengo v0.0.0-20250820003526-c297c0c1eb9d
 	k8s.io/klog/v2 v2.130.1
-	k8s.io/kube-aggregator v0.33.4
-	k8s.io/kube-openapi v0.0.0-20250318190949-c8a335a9a2ff
+	k8s.io/kube-aggregator v0.34.1
+	k8s.io/kube-openapi v0.0.0-20250710124328-f3f2b991d03b
 	k8s.io/utils v0.0.0-20250820121507-0af2bda4dd1d
 	sigs.k8s.io/yaml v1.6.0
 )
 
+replace k8s.io/apiserver => ../kubernetes-fork/staging/src/k8s.io/apiserver
```

Comment:
Obviously, this `replace` statement is only for proof of concept. I had cloned https://github.com/kubernetes/kubernetes/pull/133765 locally at this location.